modules/javafx.graphics/src/main/java/javafx/css/Declaration.java

```diff
@@ -113,6 +140,9 @@ private StyleOrigin getOrigin() {
         return true;
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
Given the `javadoc` tool bug, it's best to revert this (and have no comment on this overridden method).